internal/pkg/plugin/manager.go

```diff
@@ -59,6 +61,7 @@ func GetManager() *Manager {
 			ConfigurationManagers: make(map[string]plugin_v1.ConfigurationManager),
 			ConnectionManagers:    make(map[string]plugin_v1.ConnectionManager),
 			DebugFlag:             false,
+			ProfileFlag:					 "",
```

Comment:
funky indentation here

Reply:
in the actual code it does not look like this but I went back manually to check and reindent